frontend/ui/trapper.lua

```diff
@@ -45,69 +52,74 @@ function Trapper:wrap(func)
         -- and the above true/false won't probably be caught by
         -- any code, but let's do it anyway.
     end
-    local co = coroutine.create(func)
+    local co = coroutine.create(pcalled_func)
     return coroutine.resume(co)
 end
 
 --- Is the code wrapped by Trapper?
--- @treturn bool true if code is wrapped by Trapper, false otherwise
+--
+-- @treturn boolean true if code is wrapped by Trapper, false otherwise
 function Trapper:isWrapped()
     if coroutine.running() then
         return true
     end
     return false
 end
 
---- Allow for overriding the Paused text and buttons on the abort
--- ConfirmBox displayed when InfoMessage dismissed in Trapper:UIFeedback()
-function Trapper:setPauseText(text, ok_text, cancel_text)
-    self.paused_text = text
-    self.paused_continue_text = ok_text
-    self.paused_abort_text = cancel_text
+--- Clear any left-over widget
```

Comment:
Clears
(There's an implicit "this function": [this function] clears any left-over widgets)